app/api_routes.js

```diff
@@ -723,11 +742,31 @@ routes.get('/api/v1/streets/images/:street_id', resources.v1.street_images.get)
  *       200:
  *         description: Translations for streetmix resources. Shape depends on resource.
  */
-routes.get('/api/v1/translate/:locale_code/:resource_name', resources.v1.translate.get)
+routes.get(
+  '/api/v1/translate/:locale_code/:resource_name',
+  resources.v1.translate.get
+)
+
+/**
+ * @swagger
+ * /api/v1/flags:
+ *   get:
+ *     description: Returns a list of feature flags and its global toggle state
```

Comment:
We could possibly add an example showing the shape of the response here?

Reply:
Should it be there, or in the definitions at the top of the file? I haven't figured out how the schema works yet, as this is not just an object with properties but an object of objects ... how should I describe this?